RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.h

```diff
@@ -33,6 +33,8 @@
 #include "TObject.h"
 #include "TH1F.h"
 
+#define MAX_VERTICES 200
```

Comment:
Ok... what is the right way to do this?  Are the rules listed out somewhere?

On Wed, May 27, 2015 at 10:43 AM, Slava Krutelyov <notifications@github.com>
wrote:

> In RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.h
> <https://github.com/cms-sw/cmssw/pull/9285#discussion_r31140445>:
>
> > @@ -33,6 +33,8 @@
> >  #include "TObject.h"
> >  #include "TH1F.h"
> >
> > +#define MAX_VERTICES 200
>
> #define and using namespace in header files is against coding rules
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/9285/files#r31140445>.
>